fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/email/domain/EmailCampaign.java

```diff
@@ -192,9 +170,11 @@ public static EmailCampaign instance(final AppUser submittedBy, final Report bus
             recurrenceStartDate = null;
         }
 
-        return new EmailCampaign(campaignName, campaignType.intValue(), businessRuleId, paramValue, emailSubject, emailMessage,
-                submittedOnDate, submittedBy, stretchyReport, stretchyReportParamMap, emailAttachmentFileFormat, recurrence,
-                recurrenceStartDate);
+        return new EmailCampaign().setCampaignName(campaignName).setCampaignType(campaignType.intValue()).setBusinessRuleId(businessRuleId)
```

Comment:
I think this is missing:
this.status = EmailCampaignStatus.PENDING.getValue();

Reply:
but 'status' is not there in the previous return statement. should I add it?

Reply:
It's not but the constructor it originally called set it, can you please check?

Reply:
made changes